org/lateralgm/components/ActionList.java

```diff
@@ -150,7 +146,7 @@ public void save()
 	 * @return The frame opened or <code>null</code> if no
 	 * frame was opened.
 	 */
-	public static MDIFrame openActionFrame(MDIFrame parent, Action a)
+	public MDIFrame openActionFrame(MDIFrame parent, Action a)
```

Comment:
I missed the fact that you removed the static modifier from FRAMES; why? I thought you were just removing the static block. Anyway, if you intend to keep it that way, please rename it to just "frames."